plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergFileWriterFactory.java

```diff
@@ -132,9 +133,9 @@ public IcebergFileWriter createDataFileWriter(
         switch (fileFormat) {
             case PARQUET:
                 // TODO use metricsConfig
-                return createParquetWriter(outputPath, icebergSchema, jobConf, session, hdfsContext);
+                return createParquetWriter(MetricsConfig.getDefault(), outputPath, icebergSchema, jobConf, session, hdfsContext);
             case ORC:
-                return createOrcWriter(metricsConfig, outputPath, icebergSchema, jobConf, session, storageProperties);
+                return createOrcWriter(metricsConfig, outputPath, icebergSchema, jobConf, session, storageProperties, Optional.empty());
```

Comment:
Maybe cleaner to not have the stats limit be optional in `createOrcWriter` and instead get it from the session here, so we always have a value

Reply:
ok!